modules/bigquery/main.tf

```
@@ -38,7 +38,7 @@ resource "google_project_service" "enable_destination_api" {
 resource "google_bigquery_dataset" "dataset" {
   dataset_id = "${var.dataset_name}"
   project    = "${google_project_service.enable_destination_api.project}"
-
+  location   = "${var.location}}"
```

@kiran002 there is a typo here which appears to have caused the failure.

```suggestion
  location   = "${var.location}"
```